.github/workflows/link-check.yml

```diff
@@ -0,0 +1,22 @@
+name: check Links
+
+on:
+  push:
+    paths:
+      - 'pages/blog/posts/**/*.md'
+
+  pull_request:
+    paths:
+      - 'pages/blog/posts/**/*.md' 
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.9.0
+        with:
+          fail: true
+          args: '--base . --verbose --no-progress pages/blog/posts/**/*.md' 
```

Comment:
Next, here we would have to set base to `--base https://json-schema.org/` so that all the relative links are resolved from the base of our deployment.

Reply:
Sound good!